scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -34,6 +34,15 @@ var _moth_jump_flutter_timer: float = 0.0
 var _moth_jump_initial_height: float = 0.0
 var _is_jump_button_held: bool = false
 
+## Used for rudimentary validation of exiting the climbable while holding up.
+## Change when tiles are implemented.
+var _is_up_pressed: bool = false  
+
+@export var LAUNCH_VELOCITY: int = 100
+
+@export var CLIMB_SPEED: int = 300
+var _is_climbing: bool = false
+var _can_climb: bool = false
```

Comment:
what does this get used for?

Reply:
For climbing? Sorry I don't get what this one is asking

Reply:
It looks like one of the variables isnt used possibly?

Reply:
Oh lol, removed the extra variable